$MirOS: ports/net/curl/patches/patch-lib_http_c,v 1.1 2008/04/05 21:54:39 tg Exp $
--- lib/http.c.orig	Mon May 19 20:57:05 2008
+++ lib/http.c	Tue Aug 26 19:03:21 2008
@@ -2477,7 +2477,7 @@ CURLcode Curl_http(struct connectdata *c
              Curl_wkday[tm->tm_wday?tm->tm_wday-1:6],
              tm->tm_mday,
              Curl_month[tm->tm_mon],
-             tm->tm_year + 1900,
+             (int)(tm->tm_year + 1900),
              tm->tm_hour,
              tm->tm_min,
              tm->tm_sec);
